src/sentry/static/sentry/app/views/settings/organizationRelay/relayWrapper.tsx

```diff
@@ -145,7 +145,7 @@ class RelayWrapper extends AsyncView<Props, State> {
               icon={<IconTelescope size="xl" />}
               title={t('The middle layer between your app and Sentry!')}
               description={t(
-                'Scrub all personal information before it arrives in Sentry. Relay impove’s event reponse time and acts as a proxy for organizations that restrict HTTP communication.'
+                'Scrub all personally identifiable information before it arrives in Sentry. Relay improves event response time and acts as a proxy for organizations with restricted HTTP communication.'
```

Comment:
```suggestion
                'Scrub all personally identifiable information before it arrives into Sentry. Relay improves event response time and acts as a proxy for organizations with restricted HTTP communication.'
```

Reply:
Isn't this train jargon ("the train will arrive into central station")? I only know of these two:

- arrive in Vienna
- arrive at your place

Reply:
Yea I'm treating Sentry as a "place" for events to arrive at.

This is an alternative: 


```suggestion
                'Scrub all personally identifiable information before it arrives at Sentry. Relay improves event response time and acts as a proxy for organizations with restricted HTTP communication.'
```